core/local/steps/initial_diff.js

```diff
@@ -122,6 +130,9 @@ async function initialDiff (buffer /*: Buffer */, out /*: Buffer */, pouch /*: P
               path: was.path
             })
           }
+        } else if (foundUntouchedFile(event, was)) {
+          _.set(event, [STEP_NAME, 'md5sumReusedFrom'], was.path)
```

Comment:
Still not sure why a boolean is not sufficient.

Reply:
My bad. Since we're actually looking for corresponding metadata in multiple steps, there's no guarantee we'll get the same doc everytime. So maybe you were right and we should actually attach the doc `_id` and `_rev`... WDYT?

Reply:
`path` is actually more specific than `_id` so better keep it.

Reply:
True we could get a different doc. Which means `was` should probably be a complete `Metadata` and we should attach it.